lightning/src/ln/chan_utils.rs

```diff
@@ -1130,7 +1141,11 @@ impl CommitmentTransaction {
 		let mut txouts: Vec<(TxOut, Option<&mut HTLCOutputInCommitment>)> = Vec::new();
 
 		if to_countersignatory_value_sat > 0 {
-			let script = script_for_p2wpkh(&countersignatory_pubkeys.payment_point);
+			let script = if opt_anchors {
+			    get_to_countersignatory_with_anchors_redeemscript(&countersignatory_pubkeys.payment_point).to_v0_p2wsh()
+			} else {
+			    script_for_p2wpkh(&countersignatory_pubkeys.payment_point)
```

Comment:
maybe a good opportunity to rename this to match the `get_XXX_redeemscript` convention

Reply:
done